### PR TITLE
fix(middleware): oauth-app has no onUnhandledRequest option

### PR DESCRIPTION
**Pull request by baoshan**
This PR keeps `app.js` updated if we could remove the `onUnhandledRequest` option from `oauth-app.js`. 

Related PR: [#341](https://github.com/octokit/oauth-app.js/pull/341)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "@octokit/auth-app": "^4.0.0",
     "@octokit/auth-unauthenticated": "^3.0.0",
     "@octokit/core": "^4.0.0",
-    "@octokit/oauth-app": "^4.0.7",
+    "@octokit/oauth-app": "file:../oauth-app.js/pkg",
```
Comment (oscard0m) [inline package.json:24]:
```suggestion
    "@octokit/oauth-app": "5.0.0-beta.1",
```

**Comment by gr2m** [comment]
@baoshan is this PR ready for review / beta release?

**Comment by baoshan** [comment]
Thanks. Please review it.

I’ll open another PR once `webhooks` has `onUnhandledRequest` option removed.

**Comment by baoshan** [comment]
[This is](https://github.com/octokit/webhooks.js/pull/740) the PR to `webhooks.js`.

**Comment by wolfy1339** [comment]
This needs a final rebase

**Review by wolfy1339 — APPROVED** [review]

**Comment by baoshan** [comment]
Rebased. Depends on `"@octokit/webhooks": "11.0.0-beta.1"`.

File: README.md [hunk 2]
```diff
@@ -334,11 +334,18 @@ const app = new App({
 });
 
 const middleware = createNodeMiddleware(app);
-
-require("http").createServer(middleware).listen(3000);
+createServer(async (req, res) => {
```
Comment (gr2m) [inline README.md:337]: can you add the require statement for `createServer` on top if you remove it here?

**Review by gr2m — APPROVED** [review]
I didn't have the time to thoroughly review it and I won't be able to soon. But I don't want you to get blocked and really appreciate all your work you put into this, so I'm going to merge it into beta.

**Pull request merged**